docs/website/01_framework/dirichletBCs.md

```diff
@@ -0,0 +1,42 @@
+<!--
+SPDX-FileCopyrightText: 2022 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Dirichlet boundary conditions
+##  Introduction
+In finite element problems, it is essential to incorporate the Dirichlet boundary conditions, i.e., 
+prescribing a part of the solution to a fixed value.
+Let us consider the following:
+
+$$
+ \boldsymbol{u} = \boldsymbol{g} \quad \text{on  } \Gamma_\mathrm{D}.
+$$
+Here \(  \boldsymbol{u} \) is the solution field with a prescribed value \(\boldsymbol{g}\) on the boundary \(\Gamma_\mathrm{D}\).
+
+For the discrete algebraic problem, this translates to fixing the values of \(u_i\) to \(g_i\) in the approximation \(  \boldsymbol{u}^h = \sum_i N^i u_i \), where
+\(  N^i \) is the $i$-th ansatz function.
+
+The implementation of several functions for $\boldsymbol{g}$ is done in the class `#!cpp Ikarus::DirichletValues`.
```

Comment:
It would be better to write something as "The handling of such function $\boldsymbol{g}$ is done by the class ....